paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PaymentMethodConfigurationSettingsDefinition.kt

```diff
@@ -0,0 +1,25 @@
+package com.stripe.android.paymentsheet.example.playground.settings
+
+import com.stripe.android.paymentsheet.example.playground.model.CheckoutRequest
+
+internal object PaymentMethodConfigurationSettingsDefinition :
+    PlaygroundSettingDefinition<String>,
+    PlaygroundSettingDefinition.Saveable<String>,
+    PlaygroundSettingDefinition.Displayable<String> {
+    override val key: String = "paymentMethodConfigurationId"
+    override val displayName: String = "Payment Method Configuration ID"
+    override val defaultValue: String = ""
+    override val options: List<PlaygroundSettingDefinition.Displayable.Option<String>> = emptyList()
```

Comment:
Should we give this options instead?

Reply:
With prefilled settings for our test account? PM configs are different for each account, so I think a string is fine for now, but it might be nice to have a few settings for each test account to try different scenarios.